backend/model/container_profile.rb

```diff
@@ -27,4 +28,14 @@ def self.sequel_to_jsonmodel(objs, opts = {})
     jsons
   end
 
+
+  def validate
```

Comment:
All looks good.  Do we need to validate this at the model level as well as the JSON level?  Maybe just the JSON level is OK?